cockpit-frontend/src/components/ScrumMasterForm/ScrumMasterForm.jsx

```diff
@@ -0,0 +1,277 @@
+              id="mvpName"
+              name="mvpName"
+              label="MVP Name"
+              autoComplete="mvpName"
+              size="small"
+            />
+          </Grid>
+          <Grid item xs={12}>
+            <FormLabel className={classes.formLabel}>Enter Jira Key</FormLabel>
+
+            <TextField
+              className={classes.textField}
+              onChange={(e) => {
+                dispatch(setJiraId(e.target.value));
+                dispatch(setFormIsValid());
+              }}
+              required
+              fullWidth
+              variant="outlined"
+              id="jiraID"
+              name="jiraID"
+              label="Jira Key"
+              autoComplete="jiraID"
+              size="small"
+            />
+          </Grid>
+          <Grid item xs={5}>
+            <FormLabel className={classes.formLabel}>
+              Choose an entity
+            </FormLabel>
+
+            <FormControl
+              required
+              size="small"
+              fullWidth
+              variant="outlined"
+              className={classes.textField}
+            >
+              <InputLabel id="entityID">Entity</InputLabel>
+              <Select
+                labelId="entityID"
+                id="entity"
+                label="Entity"
+                onChange={(e) => {
+                  dispatch(setEntity(e.target.value));
+                  dispatch(setFormIsValid());
+                }}
+              >
+                <MenuItem value={1}>Entity1</MenuItem>
+                <MenuItem value={2}>Entity2</MenuItem>
+                <MenuItem value={3}>Entity3</MenuItem>
+              </Select>
+            </FormControl>
+          </Grid>
+          <Grid item xs={2} />
+          <Grid item xs={5}>
+            <FormLabel className={classes.formLabel}>Enter a cycle</FormLabel>
+
+            <TextField
+              className={classes.textField}
+              required
+              onChange={(e) => {
+                dispatch(setCycle(e.target.value));
+                dispatch(setFormIsValid());
+              }}
+              fullWidth
+              variant="outlined"
+              id="cycleID"
+              name="cycleID"
+              label="Cycle"
+              type="number"
+              autoComplete="cycleID"
+              size="small"
+            />
+          </Grid>
+          <Grid item xs={12} className={classes.center}>
+            <FormLabel>
+              You need to complete all the fields before submitting a MVP.
+            </FormLabel>
+          </Grid>
+        </Grid>{' '}
+        <div className={classes.line} />
+        <Button
+          fullWidth
+          variant="text"
+          id="submit"
+          value="Submit"
+          type="submit"
+          size="large"
+          disabled={!isFormValid}
+        >
+          Add MVP
+        </Button>
+      </form>
+    </div>
+  );
+  return (
+    <Modal
```

Comment:
Have you checked if it's possible to use the component dialog of material-ui at the place of Modal? normally with dialog we could have some cool transition animations
https://material-ui.com/components/dialogs/